src/common/stores/store-names.ts

```diff
@@ -17,4 +17,5 @@ export enum StoreNames {
     UnifiedScanResultStore,
     CardSelectionStore,
     DeviceStore,
+    ElectronWindowStateStore,
```

Comment:
I know we mention this offline but I'll coment again: I don't think we should use `Electron` in the name of our classes/enum/types. Each store is actually agnostic to where it's being used (extension or electron) and as such we should not "stigmatize" them like this. It should be up to the reader to look at the proper initializer to find out where the store (or class/type/interface/enum for that matter) is being used

Reply:
updated to remove Electron from the name in the 5th commit